open_rarity/resolver/rarity_providers/rank_resolver.py

```diff
@@ -6,5 +6,5 @@
 class RankResolver(Protocol):
     @staticmethod
     @abstractmethod
-    def get_all_ranks(contract_address: str | None, slug: str | None) -> dict[str, int]:
+    def get_all_ranks(contract_address: str) -> dict[str, int]:
```

Comment:
`contract_address_or_slug` since the name doens't matter and each kid can use either. Let's add a comment too

Reply:
updated!